core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -750,4 +751,19 @@ public FlowIdentifier modifyProxyInternal(@TenantAwareParam String crn, String p
     public FlowIdentifier rotateSecrets(@RequestObject DistroXSecretRotationRequest request) {
         return stackOperationService.rotateSecrets(request.getCrn(), request.getSecrets(), request.getExecutionType());
     }
+
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.DATAHUB_VERTICAL_SCALING)
```

Comment:
is this intentional that you are using vertical scaling permission for this feature?

Reply:
yes after discussion with the Wolf team